### ISSUE TITLE
Cannot connect to server on a Windows system

### ISSUE THREAD

**Issue by ZaLiTHkA** [NarrativeScience/Log.io#147]
I'm currently doing some research into real-time log file viewers to bundle with a couple corporate level software packages; the app suites in question are either desktop or web based Java based systems running locally with bundled Java and Tomcat on the target installation machine. So including something like Log.io would be dead simple from my side.

So far I've had no issues installing Log.io, but running it seems to be a different story.. After installing, I start the server with `log.io-server` with no issues and set the paths to my log files in `harvester.conf`, but when I try run `log.io-harvester`, I get a message stating `error: Unable to connect server, trying again...` over and over.

At first I thought it was my log files path, but I've tried the typical `E:\\path to\\log file.log` and `E:/path to/log file.log` path definitions, which made no difference. When I deliberately use an invalid path, `log.io-harvester` complains that the file cannot be found, but still no server connection.

I should note, the web server side is starting up correctly and I can access the UI through `localhost:28778`, which shows "0 streams, 0 nodes, 0 messages.." etc, with no errors or warnings shown in the browser console. Running `curl http://localhost:28777` from another prompt shows a response in the `log.io-server` window, but that's all I can do with it.

Am I being silly here? What am I missing? Is there any way to run Log.io with verbose output to see more detail?

**Comment by Deadarius** [comment]
Try to set server.host to 127.0.0.1 instead of default 0.0.0.0. It worked for me.

**Comment by ZaLiTHkA** [comment]
Thanks for the tip. I've been swamped with other work since I posted this, but hopefully I'll get a chance to look into this again soon.

**Comment by ZaLiTHkA** [comment]
I've spent most of my free time the last couple weeks working out the finer details of environment configuration for development in Windows with NodeJS. Finally got to the point where I can get native libraries to compile reliably on three different Windows machines (yay).

However, I'm still not having any joy with getting this to run under Windows. I got it running fine in my Ubuntu VM though, and then I found issue #76 which stated (semi-recently) that there are issues with Log.io and Windows. There are one or two personal projects where this would be useful to me, but I'll have to carry on looking for a real-time log tail system that doesn't commit suicide when it enters the same room as a Microsoft OS...
